sway/input/cursor.c

```diff
@@ -1271,4 +1271,33 @@ struct sway_cursor *sway_cursor_create(struct sway_seat *seat) {
 	cursor->cursor = wlr_cursor;
 
 	return cursor;
+
+}
+
+void cursor_warp_to_container(struct sway_cursor *cursor,
+								struct sway_container *container){
+	if(!container) {
+		return;
+	}
+
+	double x = 0;
+	double y = 0;
+	x = container->x + container->width / 2.0;
```

Comment:
May as well initialise `x` and `y` to these values rather than setting them to zero first.